src/test/java/test/CalculatorTest.java

```diff
@@ -0,0 +1,87 @@
+package test;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import step1.Calculator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CalculatorTest {
+    int a = 0;
+    int b = 0;
+    int c = 0;
+    int d = 0;
+    String expression = "";
+    @BeforeEach
+    void setUp() {
```

Comment:
필드와 메서드를 구분할 수 있도록 공백을 추가해주시면 가독성이 좋을 것 같아요!

```suggestion
    String expression = "";
    
    @BeforeEach
    void setUp() {
```